src/gcovr/formats/gcov/parser/text.py

```diff
@@ -237,7 +237,7 @@ def parse_metadata(
         if not line:
             continue
 
-        parsed_line = _parse_line(line, suspicious_hits_threshold)
+        parsed_line = _parse_line("", line, suspicious_hits_threshold)
```

Comment:
Would add a filename argument to the function and give the name of the gcov file here.